### PR TITLE
Added new command migrate:middleware-to-request-handler

### PR DESCRIPTION
**Pull request by michalbundyra**
Convert PSR-15 middleware to request handler in given path. It should be used to convert final middlewres binded to routes which are not using delegator/request handler.

Works only with PSR-15 middlewares (if you have interop middlewares migrate them first to PSR-15 middlewares by migrate:interop-middleware and then convert them to request handlers).

### CODE REVIEW

**Comment by weierophinney** [comment]
I would argue that this should raise an exception if it detects any `$handler->handle` calls.

**Comment by michalbundyra** [comment]
@weierophinney 
> I would argue that this should raise an exception if it detects any $handler->handle calls.

I'll try to add this check. I need to find what is the variable name and then look for `$var->handle` as you said.

File: test/MigrateMiddlewareToRequestHandler/TestAsset/expected/MyMiddlewareWithHandler.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?php
+
```
Comment (weierophinney) [inline test/MigrateMiddlewareToRequestHandler/TestAsset/expected/MyMiddlewareWithHandler.php]: We need a license docblock here and on the other test asset; we should likely also add the `declare(strict_types=1)`, though that's not completely necessary.

**Review by weierophinney — CHANGES_REQUESTED** [review]

**Comment by weierophinney** [comment]
Thanks, @webimpress!

**Pull request merged**